content/en/news/releases/1.14.x/announcing-1.14/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Announcing Istio 1.14
+linktitle: 1.14
+subtitle: Major Update
+description: Istio 1.14 release announcement.
+publishdate: 2022-05-24
+release: 1.14.0
+skip_list: true
+aliases:
+- /news/announcing-1.14
+- /news/announcing-1.14.0
+---
+
+We are pleased to announce the release of Istio 1.14!
+
+{{< relnote >}}
+
+This is the second Istio release of 2022. We would like to thank the entire Istio community
+for helping to get Istio 1.14.0 published.
+Special thanks are due to the release managers Lei Tang (Google) and Greg Hanson (solo.io),
+and to Test & Release WG lead Eric Van Norman (IBM) for his help and guidance.
+
+{{< tip >}}
+Istio 1.14.0 is officially supported on Kubernetes versions `1.21` to `1.24`.
+{{< /tip >}}
+
+Here are some of the highlights of the release:
+
+## Support for the SPIRE runtime
+
+SPIRE is a production-ready implementation of the SPIFFE specification, that offers
+pluggable multi-factor attestation and SPIFFE federation. We've made changes in the way
+we integrate with external Certificate Authorities, using the Envoy SDS API, to enable
+support for SPIRE. Thanks to the team at HP Enterprise for contributing this work!
+
+SPIRE enables the introduction of strongly attested identities through the use of a combination
+of different attestation mechanisms. SPIRE provides a variety of node and workload attestors out
+of the box for workloads running in Kubernetes, AWS, GCP, Azure, Docker, and, through a plugin
+oriented architecture, it also enables the use of custom attestors.
+SPIRE provides a pluggable integration with custom Key Management Systems, e.g. AWS KMS, for
+storing the CA private keys.
+SPIRE enables the integration with existing PKIs through the Upstream Certificate Authority plugin.
+SPIRE implements SPIFFE Federation, enabling workloads to trust peers in a different trust domain, in
+a configurable and flexible way through the Federation API.
```

Comment:
TODO rewrite "SPIRE is, SPIRE does, SPIRE etc" into a better paragraph

Reply:
@craigbox If you have suggestions on improving this paragraph, please let us know. We plan to release 1.14 around May 24.